test/unit/specs/components/Identity/createComponent.spec.js

```diff
@@ -111,7 +115,7 @@ describe("Identity::createComponent", () => {
     const response = { type: "response" };
     const result = component.lifecycle.onResponse({ response });
     expect(handleResponseForIdSyncs).toHaveBeenCalledWith(response);
-    expect(result).toBe(idSyncsPromise);
+    expectAsync(result).toBeResolvedTo(response);
```

Comment:
I think you have to do `return expectAsync`

Reply:
thank you!